docs/pages/guides/adding-delegation.mdx

```diff
@@ -0,0 +1,726 @@
+        name: "systemId",
+        type: "bytes32",
+      },
+      {
+        internalType: "uint256",
+        name: "numCalls",
+        type: "uint256",
+      },
+    ],
+    name: "initDelegation",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [],
+    name: "newCall",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+];
+
+// Create the system name constants
+
+const name2hex = (name, length) => {
+  let result = "";
+  for (let i = 0; i < name.length; i++) result += name.charCodeAt(i).toString(16);
+
+  if (length >= name.length) return result.padEnd(length * 2, "0");
+  else return result.slice(0, length * 2);
+};
+
+const systemId = (namespace, systemName) =>
+  "0x" + name2hex("sy", 2) + name2hex(namespace, 14) + name2hex(systemName, 16);
+
+const LASTCALL_SYSTEM_ID = systemId("LastCall", "LastCallSystem");
+const SYSTEMBOUND_DELEGATION = systemId("", "systembound");
+
+// A lot of code that would normally go in mud/getNetworkConfig.ts is part of the application here,
+// because `getNetworkConfig.ts` assumes you use one wallet, and here we need several.
+const networkConfig = await getNetworkConfig();
+
+const clientOptions = {
+  chain: networkConfig.chain,
+  transport: transportObserver(fallback([webSocket(), http()])),
+  pollingInterval: 1000,
+} as const satisfies ClientConfig;
+
+const publicClient = createPublicClient({
+  ...clientOptions,
+});
+
+// Create a structure with two fields:
+//   client - a wallet client that uses a random account
+//   world - a world contract object that lets us issue newCall
+const makeWorldContract = () => {
+  const client = createWalletClient({
+    ...clientOptions,
+    account: createBurnerAccount(getBurnerPrivateKey(Math.random().toString())),
+  });
+
+  return {
+    world: getContract({
+      address: networkConfig.worldAddress as Hex,
+      abi: IWorldAbi,
+      publicClient: publicClient,
+      walletClient: client,
+    }),
+    client,
+  };
+};
+
+// Create five world contracts
+const worldContracts = [1, 2, 3, 4, 5].map((x) => makeWorldContract());
+
+export const App = () => {
+  const {
+    network: { tables, useStore },
+  } = useMUD();
+
+  // Get all the calls from the records cache.
+  const calls = useStore((state) => {
+    const records = Object.values(state.getRecords(tables.LastCall));
+    records.sort((a, b) => Number(a.value.callTime - b.value.callTime));
+    return records;
+  });
+
+  // Convert timestamps to readable format
+  const twoDigit = (str) => str.toString().padStart(2, "0");
+  const timestamp2Str = (timestamp: number) => {
+    const date = new Date(timestamp * 1000);
+    return `${twoDigit(date.getHours())}:${twoDigit(date.getMinutes())}:${twoDigit(date.getSeconds())}`;
+  };
+
+  // Call newCall() on LastCall:LastCallSystem.
+  const newCall = async (worldContract) => {
+    const tx = await worldContract.write.LastCall_LastCallSystem_newCall();
```

Comment:
we need to update this with the latest version of MUD (no more system name in the function selector)